Sources/Networking/HTTPClient/HTTPResponse.swift

```diff
@@ -226,3 +231,11 @@ extension VerifiedHTTPResponse {
     }
 
 }
+
+enum HTTPResponseOrigin {
+
+    case backend
+
```

Comment:
Total nitpick but should this line be removed?

Reply:
right. According to the style guide it should. Thanks!